packages/icons/tasks/build-metadata.js

```diff
@@ -15,13 +15,21 @@ const yaml = require('js-yaml');
 
 const METADATA_PATH = path.resolve(__dirname, '../metadata.yml');
 const METADATA_OUTPUT = path.resolve(__dirname, '../metadata.json');
+const CATEGORIES_DEFINITION_PATH = path.resolve(__dirname, '../categories.yml');
 
 async function build() {
   const metadata = yaml.safeLoad(await fs.readFile(METADATA_PATH, 'utf8'));
+  const categories = yaml.safeLoad(
+    await fs.readFile(CATEGORIES_DEFINITION_PATH, 'utf8')
+  );
 
-  await fs.writeJson(METADATA_OUTPUT, metadata, {
-    spaces: 2,
-  });
+  await fs.writeJson(
+    METADATA_OUTPUT,
+    { ...metadata, ...categories },
```

Comment:
```suggestion
    { ...metadata, categories },
```

Reply:
@joshblack would this cause the data to nest differently?

Reply:
@joshblack yeah that seems to add to an extra level of duplicate nesting:
![Screen Shot 2019-07-22 at 5 41 54 PM](https://user-images.githubusercontent.com/3846271/61670105-16e37780-aca8-11e9-9b6b-c2b8b559b1a5.png)

Reply:
Got it! Sorry, assumed `categories` was a different structure, looks great 👍